db/seeds.rb

```diff
@@ -1,6 +1,33 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
 #
-10.times do |i|
-  User.create(email: "test#{i}@email.com", password: "password#{i}")
+require "httpclient"
+require "csv"
+
+# 10.times do |i|
+#   User.create(email: "test#{i}@email.com", password: "password#{i}")
+# end
+
+STAR_COLUMNS = ["pl_hostname","ra","dec","st_dist","st_teff","st_mass","st_rad","st_spstr","st_logg","st_lum","st_dens","st_age"]
```

Comment:
Line is too long. [129/80]<br>Space missing after comma.<br>Use `%w` or `%W` for array of words.